### PR TITLE
Enable Mutation Assessor and remove unavailable fields

### PR DESCRIPTION
**Pull request by leexgh**
Part of: https://github.com/genome-nexus/genome-nexus/issues/634
Test: https://deploy-preview-138--genome-nexus-frontend.netlify.app/variant/17:g.7577121G%3EA
image

### CODE REVIEW

File: src/page/VariantStore.ts [hunk 1]
```diff
@@ -86,6 +86,13 @@ export class VariantStore {
     @observable public variant: string = '';
     @observable public selectedTranscript: string = '';
 
+    readonly genomeBuild: MobxPromise<string> = remoteData({
+        await: () => [this.mainStore.genomeBuild],
+        invoke: async () => {
+            return Promise.resolve(this.mainStore.genomeBuild.result);
+        },
+    });
```
Comment (onursumer) [inline src/page/VariantStore.ts:94]: Should we just do this?

```suggestion
    readonly genomeBuild: MobxPromise<string> = this.mainStore.genomeBuild;
```

if not maybe this?

```suggestion
    readonly genomeBuild: MobxPromise<string> = remoteData({
        await: () => [this.mainStore.genomeBuild],
        invoke: () => Promise.resolve(this.mainStore.genomeBuild.result),
    });
```

**Review by onursumer — COMMENTED** [review]
Looks good! Added a minor question about the genome build promise.

File: src/page/VariantStore.ts (hunk 1 above)
Comment (leexgh) [reply src/page/VariantStore.ts]: @onursumer 👍

**Review by inodb — APPROVED** [review]
LGTM! Thanks so much!

**Pull request merged**